### ISSUE TITLE
Update new accessibilityFeature vocabulary list with ARIA

### ISSUE THREAD

**Issue by clapierre** [daisy/ace#369]
We have deprecated the use of accessibilityAPI to include "ARIA" and have moved this to accessibilityFeature instead.

Currently Ace reports this unknown value as a "moderate" error.
Unknown values not in our current vocabulary list should be marked as informational as the lists aren’t restricted to the vocabulary.

**Comment by danielweck** [comment]
Fixed via https://github.com/daisy/ace/commit/327f267172d1ae0d9feb0db6a8f7df573a631588

**Comment by clapierre** [comment]
Yes I think it should report these as minor violations because the author usually has made a typo say in accessMode instead of "auditory" they used "audio".
So I think this should go beyond just accessibilityFeature and should be included for
accessMode
accessModeSufficient
accessibilityHazard
as well as 
accessibilityFeature

Now conformsTo has changed from a URL now to a default set of tags, but I believe we still allow for the URL approach so that might be more difficult to check, and as new HTML specifications get released we probably don't want to have to keep updating this list of valid tags but I will let others weigh in on that debate.  @mattgarrish thoughts?
